.github/workflows/release.yaml

```diff
@@ -1,11 +1,14 @@
 name: "Prepare the release"
 on:
-  push:
-    tags: [ 'v*' ]
+  workflow_run:
+    workflows: [Publish container images]
+    types:
```

Comment:
I think right now this would publish a release on every commit to main. The container publish runs on any tag and any commit to main. Is there a way we can only run this on a tag?

Reply:
Good catch. And unfortunately, it doesn't look like we can.

Looks like properly solving this involves creating a reusable workflow for publishing our images and creating separate workflows for publishing dev and release images, making use of that reusable workflow. Then we can trigger based only on release images. Or just have a single release workflow that does everything.

Reply:
i think consolidating to a single workflow makes sense to me. We would then just conditionally enable the actual publish part if its a tag.